samples/Azure.ResourceManager.Sample/Generated/Models/ProximityPlacementGroupPatch.Serialization.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Azure.ResourceManager.Sample.Models
 {
-    internal partial class ProximityPlacementGroupPatch : IUtf8JsonSerializable
+    public partial class ProximityPlacementGroupPatch : IUtf8JsonSerializable
```

Comment:
Why did we change serialization/deserialization class to public?

Reply:
this model is now used on a public API so it became public.  We have an internalizer that runs and hides all models that aren't publicly exposed to minimize the public api footprint but after this change this model is no longer internal

Reply:
Not in this case but we also generate deserialization method in this class, so it will also be public?

Reply:
And we have [ProximityPlacementGroupPatch](https://github.com/Azure/autorest.csharp/blob/7f88dec51a06eff8dc3780d299441904116a64d6/samples/Azure.ResourceManager.Sample/Generated/Models/ProximityPlacementGroupPatch.cs#L11) public.

Reply:
I was talking about [this](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestProjects/NameConflicts/Generated/Models/HttpMessage.Serialization.cs#L26) case but nm we generate an internal deserialization method.